docs/src/components/Header/styles.module.css

```diff
@@ -0,0 +1,139 @@
+.navItem {
+  list-style-type: none;
+  padding-left: 16px;
+  padding-right: 16px;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 500;
+  display: flex;
+}
+
+.navItem:last-child {
+  padding-right: 0px;
+}
+
+.navLink {
+  color: #3b4473;
+  text-decoration: none;
+}
+
+.navLink:hover {
+  text-decoration: none;
+}
+
+.headerOuterWrapper {
+  position: relative;
+  z-index: 1000;
+  margin-bottom: 65px;
+  background-color: #ffffff;
+  border-bottom: 1px solid #aecee5;
```

Comment:
can we get this color from color file?

Reply:
This is individual css module. currently `scss` and `css module` both are being used.